splinterd/api/static/openapi.yml

```diff
@@ -327,6 +327,145 @@ paths:
         500:
           description: Internal service error
 
+  /user/notifications:
+    get:
+      description: Retrieve a list of all notifications
+      responses:
+        200:
+          description: Notifications have been retrieved
+          content:
+            application/json:
+              schema:
+                type: array
+                $ref: '#/components/schemas/Notification'
+        403:
+          description: User not authorized to access this endpoint
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    post:
+      description: Send a list of notifications to the user module
```

Comment:
`user module` is an internal thing, I don't think it should be mentioned in this doc. I don't think it would is clear for the consumers of this doc what `user module` is.

Reply:
updated